gtfs-realtime/CHANGES.md

```diff
@@ -18,7 +18,7 @@ The official specification, reference and documentation are written in English.
   - Anyone is allowed to vote yes/no in a form of comment to the pull request, and votes can be changed until the end of the voting period.
     If a voter changes her vote, it is recommended to do it by updating the original vote comment by striking through the vote and writing the new vote.
   - Votes before the start of the voting period are not considered.
-1. The proposal is accepted if there is a consensus yes with at least 3 votes.
+1. The proposal is accepted if there is a unanimous consensus yes with at least 3 votes.
```

Comment:
[According to Google](https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&es_th=1&ie=UTF-8#q=unanimous) it is "a unanimous vote".  So, "a" is correct.  "yes" is somewhat redundant, but I left it in to be clear (versus a "consensus no", I guess :) ).